### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by KyumKyum** [KyumKyum/gamechatban.github.io#1]
![argue](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/2b37b8d5-8ccc-4196-8383-a9d39433ea0e)

<img width="658" alt="tf_idf_df" src="https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/920ab174-c1bb-46ab-a1bd-305df5fef27e">

<img width="557" alt="tf_idf_low" src="https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/3aba0e8c-0ba9-42af-a1c3-0113efed3dfb">

<img width="658" alt="tf_idf_high" src="https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/87111630-63d9-4585-8570-6e51219918fc">

![TF_IDF  Scatter Plot](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/4ba98d24-b777-44d7-9cec-1fa05b4a8c06)

![TF_IDF  Word Cloud](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/0225d1c5-cd98-4dad-9d2b-42a34fc33b4b)

![LSTM_Graphical Result](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/977e0eb3-00ed-40a7-a3ea-c6cc0d2cd839)

![LSTM-Training and Validation Loss](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/3e4e990e-4511-4955-8d69-b49bb5d2c72a)

![LightGBM: QQPlot](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/57d32da5-6ed6-4c59-ba9e-e3788e62bb4a)

<img width="516" alt="q-q" src="https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/6bef0e79-1ee2-49ff-9d0b-77ca88b5a7fb">

![Heatmap (improved)](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/a5326c3d-fb4f-46dd-8f81-6fa1c5a5225f)

<img width="480" alt="Cfm" src="https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/6c89d411-2bb3-40b7-8469-16ce21568406">

![ROC Curve](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/043588be-5ffe-4cc6-b1d2-84bf054e61d7)

![raw](https://github.com/KyumKyum/gamechatban.github.io/assets/59195630/09ec0c53-1ca5-4190-a23f-8b9c0539085b)
